app/src/main/java/com/example/firstandroid/MainActivity.kt

```diff
@@ -51,7 +58,7 @@ class MainActivity : ComponentActivity() {
                 setContent {
                     val listOfBathroomData = response.body()?.data
                     var isDialogOpen by remember { mutableStateOf(false) }
-
```

Comment:
Ways to handle state across compositions:

1. Remember the state in the Composable. The state will get wiped if the app is put in the background and OS needs to reclaim resources.

2. Keep the state in the viewmodel and pass it to the Composable. This state will survive even if the app is destroyed by the OS.